<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QDialog" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblVideoUrl">
     <property name="text">
      <string>Video Url</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtVideoUrl">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>Enter video url here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblSaveFolder">
     <property name="text">
      <string>Save Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="txtSaveFolder">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Select save folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSelectSaveFolder">
       <property name="toolTip">
        <string>Select Save Folder</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblFileFormat">
     <property name="text">
      <string>File Format</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cmbFileFormat">
     <property name="placeholderText">
      <string>Select file format</string>
     </property>
     <item>
      <property name="text">
       <string>MP4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WEBM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MP3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OPUS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FLAC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WAV</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblNewFIlename">
     <property name="text">
      <string>New Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtNewFilename">
     <property name="placeholderText">
      <string>Enter new filename here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblError">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(250, 71, 55); }</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutBtns">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
